OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -168,6 +168,16 @@ public virtual void Tick(Actor self)
 			delayedActions.RemoveAll(a => a.First <= 0);
 		}
 
+		void INotifyCreated.Created(Actor self)
+		{
+			Initialize(self);
```

Comment:
It would be worth a comment here explaining that the implementation is split into a protected method to allow subclassing.